src/services/login.ts

```diff
@@ -1,26 +1,52 @@
 import boom from '@hapi/boom';
+import jwt from 'jsonwebtoken';
 import bcrypt from 'bcrypt';
 import userData from '../data-access/users';
-import { UserDB } from '../types';
+import { UserDB, LoggedInUser } from '../types';
 
 
 const verifyLoginDetails = async function (email: string, password: string): Promise<UserDB> {
   const result = await userData.getUserByEmail(email);
```

Comment:
Just one issue on line 9 of `login.ts`, a change to `getByEmail(email)` will make it work.

Reply:
I have made, comitted and pushed the fix.